server/planning/planning/planning_post.py

```diff
@@ -97,12 +98,20 @@ def post_planning(self, plan, new_post_state):
         get_resource_service('planning').update(plan['_id'], updates, plan)
         get_resource_service('planning_history')._save_history(plan, updates, 'post')
 
-        # enqueue the planning item
-        # these fields are set for enqueue process to work. otherwise not needed
-        plan.setdefault(config.VERSION, 1)
+        # Set a version number
+        version = int((datetime.utcnow() - datetime.min).total_seconds() * 100000.0)
```

Comment:
have `version` in the history so that we can show the delivery in the history.

Reply:
Yep